### DISCUSSION TITLE
Error while the synthesized bit

### DISCUSSION THREAD

**Discussion by WINNIEPUPU** [Xilinx/finn#941] [Q&A] [answered]
Hi, everyone! I'm running finn example from the deployment of LPYOLO for end-to-end deployment,it shows the error that a step synthesis bitfile appeared in "the synthesized bit [15/17]"
I am attaching the screenshot of the error.
![错误3](https://github.com/Xilinx/finn/assets/149388714/048404be-aa9c-4f0f-9349-76c97eaf182e)
![错误4](https://github.com/Xilinx/finn/assets/149388714/c313eae5-722e-4247-bd9a-11e971fdbaab)
![错误5](https://github.com/Xilinx/finn/assets/149388714/d3e06673-2035-40ad-b53a-8284c7020a38)

I am using Ubuntu 20.04, Xilinx tools version: 2020.1.  And finn:0.7. 
Can someone help me? I'm new with FINN.

**Comment by iksnagreb** [comment, marked as answer]
Did you check the log at the path specified in the exception message? What does it say? There should probably be a few lines starting with `ERROR:` towards the end.

**Reply by WINNIEPUPU** [reply]
Thank you very much for your reply. We have reviewed the VIVADO log file under the error path and found that the error is due to insufficient board resources. We have resolved this issue by adjusting the parallelism setting in the JSON file. Finally, we appreciate your help.
